charts/service/templates/ingressroute.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app.kubernetes.io/instance: traefik
+    app.kubernetes.io/name: traefik
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - kind: Rule
+      match: "Host(`{{ .Release.Name }}.{{ .Values.env }}.allstar.dev`){{ range $hostname := .Values.ingress.hostnames }} || Host(`{{ $hostname }}`){{ end }}"
+      services:
+        - name: {{ .Release.Name }}
+          port: 3000
```

Comment:
Should this port be hardcoded to 3000 or should it be getting that value from a values file?

Reply:
Good catch, should def. be coming from `.Values.port`.